src/traversal.js

```diff
@@ -1,19 +1,24 @@
 import { base58btc } from './bases/base58.js'
 
 /**
- * @typedef {import('./cid.js').CID} CID
+ * @template [C=number] - multicodec code corresponding to codec used to encode the block
+ * @template [A=number] - multicodec code corresponding to the hashing algorithm used in CID creation.
+ * @template [V=0|1] - CID version
+ * @typedef {import('./cid').CID<unknown, C, A, V>} CID
  */
 
 /**
- * @template T
- * @typedef {import('./block.js').Block<T>} Block
+ * @template [T=unknown] - Logical type of the data encoded in the block
+ * @template [C=number] - multicodec code corresponding to codec used to encode the block
+ * @template [A=number] - multicodec code corresponding to the hashing algorithm used in CID creation.
+ * @template [V=0|1] - CID version
+ * @typedef {import('./block/interface').BlockView<T, C, A, V>} BlockView
  */
 
 /**
- * @template T
  * @param {Object} options
  * @param {CID} options.cid
```

Comment:
if we're going to bump semver-major anyway (I think we should), then maybe we should also make `CID` go away in here and make this param a `Link` and the `load` param take a `Link` too?

Reply:
For internal stuff I don’t think it really matters & doesn’t seem worth breaking pplz code.